### DISCUSSION TITLE
求助 mybatis连接doris @Transactional 失效

### DISCUSSION THREAD

**Discussion by HJHHHHHHHHH** [apache/doris#10299] [A - General / Q&A]
在Navicat中 我用
begin;
insert ~~~~
rollback;
是可以成功回滚的，代码里面就不行了。求助

**Comment by dataalive** [comment]
你是想实现插入数据的事务？这个是doris天生支持的，不需要依赖这个begin和rollback。

**Comment by hf200012** [comment]
> 在Navicat中 我用 begin; insert ~~~~ rollback; 是可以成功回滚的，代码里面就不行了。求助

doris 不支持 事物回滚
